home.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+  <title>Senior Swift - Concise weekly blog covering skills you need day to day that employers want</title>
+  <link rel="stylesheet" href="styles.css">
+
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Playball|Playfair+Display:700" rel="stylesheet">
+</head>
+
+<body class="home-page">
+  <section class="home-banner">
+    <h1 class="blog-title">Swifty</h1>
+    <h2 class="blog-description">Weekly swift articles covering actual useful skills with short concise posts</h2>
+  </section>
+
+  <section class="blog-post-list">
```

Comment:
section should have a header

Reply:
..sorry, heading. h2, h3 etc.
Though your page should also have a <header> element! And a <footer> element.

Reply:
"...should also have a 'header' element! And a 'footer' element".
I wrote the tags, but text inside lesser-than and greater-than signs seem to be ignored...

Reply:
You can use \` to surround code snippets like this:  `<header>`

Reply:
Hmm I'd like to just show the posts immediately. I've noticed other sites often have a header saying "Recent Posts" etc but I feel it speaks for itself.

Do I need to do this for the sake of page scraping/ SEO?

Reply:
> "...should also have a 'header' element! And a 'footer' element".

I've converted HomeBanner from a `<div>` to a `<header>`. The footer I'm going to do in a future branch as I plan on putting quite a lot of content in it. (social media, git, Stackoverflow links, and a photo). I figure a detailed footer is more effective than an about page no-one will ever click on.